Doc/library/mailbox.rst

```diff
@@ -587,12 +587,27 @@ Supported mailbox formats are Maildir, mbox, MH, Babyl, and MMDF.
    remarks:
 
 
-   .. method:: get_file(key)
+   .. method:: get_bytes(key, from_=False)
+
+      Note: This method has an extra parameter (*from_*) compared with other classes.
```

Comment:
I don't know if it's better to repeat the same note or just say something like "See XXX for the meaning of `from_`". It's fine to repeat it twice, one for `mbox` and one for `MMDF`, but for the methods just after, I think it's fine to mention the first documented method of the sub-section. WDYT?

Reply:
Let's keep the current wording.